SwiftSyft/Classes/SyftWebSocket.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  SyftWebSocket.swift
+//  SwiftSyft
+//
+//  Created by Sasha Bataieva on 25/01/2020.
+//
+
+import Foundation
+import Starscream
+
+public enum WSEventType {
+    case connected
+    case disconnected
+    case text(String)
+    case binary(Data)
+    case error(Error?)
+    case cancelled
+}
+
+public class SyftWebSocket: SocketClientProtocol {
```

Comment:
`SyftWebSocket` and `SyftWebSocketIOS12` feels redundant. We can just use `SyftWebSocketIOS12` directly. If you added `WSEventType` because you feel limited by the number of methods in `SocketClientProtocol`, we can just add `sendText(text: String)` to `SocketClientProtocol`. There are also unused cases in `WSEventType` like `error` and `cancelled`.

Adding `sendText(text: String)` to `SocketClientProtocol` eliminates the need for `SyftWebSocket` altogether. We can just use `SyftWebSocketIOS12`.

If you're confused how it will work in the signalling client, it will work kind of like this:

```
var socketClient: SyftClientProtocol  = {
    if (iOS 13) {
       return SyftWebSocketiOS13()
   } {
       return SyftWebSocketiOS12()
  }

}()
```

Reply:
ok